packages/shared/voiceChat/handlers.ts

```diff
@@ -1,26 +1,37 @@
 import { getCurrentUserProfile } from '../profiles/selectors'
 import { Package, VoiceFragment } from '../comms/interface/types'
-import { setupPeer } from '../comms/peers'
+import { getPeer } from '../comms/peers'
 import { store } from '../store/isolatedStore'
 import { getVoiceHandler, shouldPlayVoice } from './selectors'
 import { voiceChatLogger } from './context'
+import { trackEvent } from 'shared/analytics'
 
 export function processVoiceFragment(message: Package<VoiceFragment>) {
   const state = store.getState()
   const voiceHandler = getVoiceHandler(state)
   const profile = getCurrentUserProfile(state)
```

Comment:
Is this necessary?

Reply:
This code is from a previous voice chat. I tried to modify it least as possible.